src/App.jsx

```diff
@@ -4,13 +4,28 @@ import {
   Link,
 } from 'react-router-dom';
 
-import HomePage from './HomePage';
-import AboutPage from './AboutPage';
-import RestaurantsPage from './RestaurantsPage';
-import RestaurantPage from './RestaurantPage';
-import NotFoundPage from './NotFoundPage';
+import { useDispatch } from 'react-redux';
+
+import HomePage from './pages/HomePage';
+import AboutPage from './pages/AboutPage';
+import RestaurantsPage from './pages/RestaurantsPage';
+import RestaurantPage from './pages/RestaurantPage';
+import NotFoundPage from './pages/NotFoundPage';
+import LoginPage from './pages/LoginPage';
+
+import { loadItem } from './services/storage';
+
+import { setAccessToken } from './modules/actions';
 
 export default function App() {
+  const dispatch = useDispatch();
+
+  const accessToken = loadItem('accessToken');
+
+  if (accessToken) {
+    dispatch(setAccessToken({ accessToken }));
+  }
```

Comment:
이 둘은 같이 쓰이니, 빈 줄이 없어야 할 것 같아요
```suggestion
  const accessToken = loadItem('accessToken');
  if (accessToken) {
    dispatch(setAccessToken({ accessToken }));
  }
```